desc/objectives/_geometry.py

```diff
@@ -523,7 +511,7 @@ def build(self, eq=None, use_jit=True, verbose=1):
             Level of output.
 
         """
-        eq = eq or self._eq
+        eq = self.things[0]
```

Comment:
Is this enforced somewhere or assumed?

Reply:
Enforced when we call `super().__init__` and specify `things`